server/backend/database/client_info.go

```diff
@@ -80,7 +81,13 @@ type ClientInfo struct {
 // CheckIfInProject checks if the client is in the project.
 func (i ClientInfo) CheckIfInProject(projectID types.ID) error {
 	if i.ProjectID != projectID {
-		return ErrClientNotFound
+		return fmt.Errorf(
+			"client not in project: clientID: %s, client's projectID: %s, target projectID: %s, %w",
+			i.ID.String(),
+			i.ProjectID.String(),
+			projectID.String(),
+			ErrClientNotFound,
+		)
```

Comment:
```suggestion
		return fmt.Errorf(
			"check client(%s,%s) in project(%s): %w",
			i.ID.String(),
			i.ProjectID.String(),
			projectID.String(),
			ErrClientNotFound,
		)
```

It would be nice to express the error message more concisely.

We were writing code according to the uber style guide: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping